docs/reference/content/guides/querying.md

```diff
@@ -0,0 +1,250 @@
+course, the value to use when applying the `condition` clause.  The `condition` parameter is a bit more complicated.  At its simplest, 
+the condition is just a field name.  In this case, the condition is assumed to be an [equality]({{< docsref "reference/operator/query/eq/"
+ >}}) check.  There is a slightly more complicated variant, however.
+ 
+The `condition` value can also contain an operator.  For example, to compare a numeric field against a value, you might write something 
+like this:
+
+```java
+query.filter("price >=", 1000);
+```
+
+In this case, we're instructing Morphia to add a filter using [$gte]({{< docsref "reference/operator/query/gte/" >}}).  This would result
+ in a query that looks like this:
+ 
+```javascript
+{ price: { $gte: 1000 } }
+```
+
+The list of supported filter operations can be found in the [FilterOperator]({{< srcref
+ "morphia/src/main/java/org/mongodb/morphia/query/FilterOperator.java">}}) class.  Each filter operator can either be referenced by its 
+ Mongodb "dollar name" or by the aliases listed afterward.  For example, with the `EQUAL` operator you can use the canonical `$eq` operator
+  as you would when building a query in the shell, or you could opt to use either `=` or `==` which might feel a little more natural to 
+  use than the dollar operators.
+ 
+### `field()`
+
+For those who would prefer more compile time validation of their queries, there is `field()`.  This method takes only the field name and 
+returns an instance of a [class]({{< srcref "morphia/src/main/java/org/mongodb/morphia/query/FieldEnd.java">}}) providing methods with which
+ to define your filters.  This approach is slightly more verbose but can be validated by the compiler to a much greater degree than 
+ `filter()` can be.  To perform the same query as above, you'd write this:
+ 
+```java
+query.field("price").greaterThanOrEq(1000);
+```
+
+This results in the exact same query as the `filter()` version but has the advantage that any typo in the operation name (method in this 
+case) would easily be caught by an IDE or compiler.  Which version you use is largely a question of preference.
+
+{{% note %}}
+Regardless of the approach used, the field name given can be either the Java field name or the document field name as defined by the 
+`@Property` annotation on the field.  Morphia will normalize the name and validate the name such that a query with a bad field name will 
+result in an error.
+{{% /note %}}
+
+## Complex Queries
+
+Of course, queries are usually more complex than single field comparisons.  Morphia offers both `and()` and `or()` to build up more 
+complex queries.  An `and` query might look something like this:
+
+```java
+q.and(
+    q.criteria("width").equal(10),
+    q.criteria("height").equal(1)
+);
+```
+
+An `or` clause looks exactly the same except for using `or()` instead of `and()`, of course.  For these clauses we use the `criteria()` 
```

Comment:
Historical reasons.  I'm not sure really other than field().someOp() returns a Query and I don't think the builder is quite smart enough to make that transition.  Yet, anyway.  I have ideas...